xarray/core/accessor_dt.py

```diff
@@ -31,6 +31,10 @@ def _access_through_cftimeindex(values, name):
     if name == "season":
         months = values_as_cftimeindex.month
         field_values = _season_from_months(months)
+    elif name == "date":
+        raise AttributeError(
+            "Attribute `date` is not available for CFTimeIndex. Consider using `floor('D')` instead."
```

Comment:
Is that `time.floor("D")` or `time.dt.floor("D")`?

Reply:
Thanks @mathause for pointing me to the test locations. 

It is `time.dt.floor("D")`, where `time.dt` can be a `CFTimeIndex`.

Reply:
Could we add this info here i.e. "Consider using `floor('D')` instead." --> "Consider using the floor method instead, for instance: `.time.dt.floor('D')` "?